.github/workflows/ci-build.yml

```
@@ -11,9 +11,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        docker_db_version: [ 7.0.19, 7.1.11 ]
+        docker_db_version: [ 7.0.20, 7.1.17 ]
```

```suggestion
        docker_db_version: [ 7.1.17 ]
```
We could remove 7.0 because it will be end-of-life soon.

---

Let's wait with that until the official EoL.